Branch-SDK/Branch-SDK/BranchEvent.h

```diff
@@ -10,7 +10,7 @@
 #import "BNCCommerceEvent.h"
 #import "BranchUniversalObject.h"
 
-///@functiongroup Branch Event Logging
+///@group Branch Event Logging
```

Comment:
Is Appledoc for this SDK ever generated and stored/hosted anywhere? It would make a good SDK reference.

Reply:
Yes -- I have an on going project that creates a doxygen doc and publishes it in Dash and as a docset.  Appledoc seems to have been abandoned.

The doc isn't ready yet -- There's a lot of work editing work needed to make the document worth publishing.